quickwit-metastore/src/tests.rs

```diff
@@ -1997,119 +1991,54 @@ macro_rules! metastore_test_suite {
 
             #[tokio::test]
             async fn test_metastore_replace_splits() {
+                let _ = tracing_subscriber::fmt::try_init();
                 crate::tests::test_suite::test_metastore_replace_splits::<$metastore_type>().await;
             }
 
             #[tokio::test]
             async fn test_metastore_mark_splits_for_deletion() {
-                crate::tests::test_suite::test_metastore_mark_splits_for_deletion::<$metastore_type>(
-                )
+                let _ = tracing_subscriber::fmt::try_init();
+                crate::tests::test_suite::test_metastore_mark_splits_for_deletion::<$metastore_type>()
                 .await;
             }
 
             #[tokio::test]
             async fn test_metastore_delete_splits() {
+                let _ = tracing_subscriber::fmt::try_init();
                 crate::tests::test_suite::test_metastore_delete_splits::<$metastore_type>().await;
             }
 
             #[tokio::test]
             async fn test_metastore_list_all_splits() {
+                let _ = tracing_subscriber::fmt::try_init();
                 crate::tests::test_suite::test_metastore_list_all_splits::<$metastore_type>().await;
             }
 
             #[tokio::test]
             async fn test_metastore_list_splits() {
+                let _ = tracing_subscriber::fmt::try_init();
                 crate::tests::test_suite::test_metastore_list_splits::<$metastore_type>().await;
             }
 
             #[tokio::test]
             async fn test_metastore_split_update_timestamp() {
+                let _ = tracing_subscriber::fmt::try_init();
                 crate::tests::test_suite::test_metastore_split_update_timestamp::<$metastore_type>(
                 )
                 .await;
             }
 
             #[tokio::test]
-            async fn test_metastore_add_source() -> anyhow::Result<()> {
-                crate::tests::test_suite::test_metastore_add_source::<$metastore_type>().await
+            async fn test_metastore_add_source() {
+                let _ = tracing_subscriber::fmt::try_init();
+                crate::tests::test_suite::test_metastore_add_source::<$metastore_type>().await;
             }
 
             #[tokio::test]
             async fn test_metastore_delete_source() {
+                let _ = tracing_subscriber::fmt::try_init();
                 crate::tests::test_suite::test_metastore_delete_source::<$metastore_type>().await;
             }
         }
-    };
-}
-
-#[cfg(feature = "postgres")]
-macro_rules! metastore_test_suite_for_postgresql {
```

Comment:
I have no clue why this thing exists.
As a result 2 unit tests were not running on postgresql. (add_source and delete_source).